---
title: "Semana 22: Probando HumanDetection"
categories:
  - Weblog
tags:
  - DeepLearning
---

---
El objetivo de esta semana era probar el ejercicio ya hecho de Human Detection de RoboticsAcademy. En primer lugar he seguido las instrucciones para realizar el "Fine Tuning" de la red ya entrenada para que detecte a personas. Al intentar ejecutar el script para reentrenar la red da errores por la incompatibilidad de las dimensiones del dataset y el objeto iterador que tiene. 

He intentado cambiar el código para solucionar este problema sin mucho éxito. Finalmente he buscado el repositorio original del proyecto y he encontrado que se trata de una issue no resuelta [Issue 168](https://github.com/qfgaohao/pytorch-ssd/issues/168). La única solución que se plantea es hacer downgrade de pytorch a la versión 1.0.0, pero tras haberlo hecho el error persiste.

Finalmente he intentado probar el modelo de prueba que ya se encuentra entrenado. En este caso el modelo no se llega a cargar y por lo tanto no me es posible probarlo:
![](/2022-tfg-alejandro-moncalvillo/images/humandetection1.png)
